### PR TITLE
Blank Constructor

### PR DESCRIPTION
**Pull request by kammitama5**

### CODE REVIEW

File: codeworld-api/src/CodeWorld/Driver.hs [hunk 1]
```diff
@@ -691,6 +693,7 @@ getPictureCS (Sector cs _ _ _) = cs
 getPictureCS (Arc cs _ _ _) = cs
 getPictureCS (ThickArc cs _ _ _ _) = cs
 getPictureCS (Text cs _) = cs
+getPictureCS (Blank _) = emptyCallStack
```
Comment (cdsmith) [inline codeworld-api/src/CodeWorld/Driver.hs]: You have a call stack here.  I'm not sure why the original code doesn't have a CallStack for `Pictures`, but I think you should go ahead and produce the one you've got.

File: codeworld-api/src/CodeWorld/Picture.hs [hunk 2]
```diff
@@ -108,6 +108,8 @@ data Picture
     | CoordinatePlane CallStack
     | Logo CallStack
     | Pictures [Picture]
+    | Blank Picture
```
Comment (cdsmith) [inline codeworld-api/src/CodeWorld/Picture.hs]: Update `blank` below to  this?

    blank :: HasCallStack => Picture
    blank = Blank callStack

File: codeworld-api/src/CodeWorld/Picture.hs [hunk 3]
```diff
@@ -108,6 +108,8 @@ data Picture
     | CoordinatePlane CallStack
     | Logo CallStack
     | Pictures [Picture]
+    | Blank CallStack
+
```
Comment (cdsmith) [inline codeworld-api/src/CodeWorld/Picture.hs]: No need for a second blank line here.

Comment (kammitama5) [reply codeworld-api/src/CodeWorld/Picture.hs]: Thanks!

File: codeworld-api/src/CodeWorld/Driver.hs (hunk 1 above)
Comment (cdsmith) [reply codeworld-api/src/CodeWorld/Driver.hs]: Were you going to return the stored call stack here?

Comment (kammitama5) [reply codeworld-api/src/CodeWorld/Driver.hs]: sorry. I had initially compared it to Pictures, so was using emptyCallStack to get it to type check :/

**Pull request merged**